src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
+  public static void main(String[] args) {
+    Truck truck = new Truck();
+    Excavator excavator = new Excavator();
+    Bulldozer bulldozer = new Bulldozer();
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop): Use abstract reference instead of specific one where possible: